components/src/molecules/tabs/Tabs.js

```diff
@@ -22,8 +24,43 @@ export default {
     selectTab (selectedTab) {
       this.$emit('click', selectedTab.name)
       this.tabs.forEach(tab => {
-        tab.isActive = (tab.name === selectedTab.name)
+        if (selectedTab === this.activeFocusedTab) {
+          tab.isActive = tab.name === this.tabs[this.activeFocusedTab].name
+        } else {
+          tab.isActive = tab.name === selectedTab.name
+        }
       })
+    },
+    focus (el) {
+      this.$refs[el][0].focus()
+    },
+    handleEvt (evt) {
+      const key = evt.keyCode
+
+      if (key === KeyCodes.TAB) {
+        evt.preventDefault()
+        evt.stopPropagation()
+        this.$refs.content.focus()
+      }
+
+      if ((key === KeyCodes.LEFT || key === KeyCodes.UP || key === KeyCodes.HOME) && this.activeFocusedTab > 0) {
```

Comment:
my only concern: shouldn't we use here `else if` instead of separate `if` condition?